src/studio/src/designer/frontend/packages/schema-editor/src/components/TopToolbar.module.css

```diff
@@ -0,0 +1,29 @@
+.toolbar {
+  background: #fff;
+  border-bottom: 1px solid #c9c9c9;
+  display: flex;
+  padding: 8px;
+}
+
+.toolbar > *, .toolbar > :global(.MuiGrid-item) {
+  margin-right: 1rem;
+}
+
+.toolbar button {
+  color: #0062ba;
+  transition: none;
+}
+
+.toolbar button[class*="selected"] {
+  color: #fff;
+}
+
+.toolbar button[class*="toggle"] {
+  font-size: 1em;
+  padding-top: 4px;
+}
```

Comment:
Kunne ikke dette også vært egne klasser? Altså typisk toolbar-button--selected og toolbar-button-toggle?

Reply:
Ikke min kode opprinnelig, men det ser ut til at dette er gjort for å finne en automatisk generert klasse fra designsystemet.